File: message-api-schema-validator/src/main/java/uk/ac/jisc/rdss/tests/term/create/TermCreateRequestTest.java
package uk.ac.jisc.rdss.tests.term.create;

import org.junit.Test;

import uk.ac.jisc.rdss.tests.AbstractSchemaValidatorTest;

public class TermCreateRequestTest extends AbstractSchemaValidatorTest {

    @Override
    protected String getSchemaFileName() {
        return "messages/term/create/request_schema.json";
    }

    @Test
    public void validateRequest() throws Exception {
        validateJson("messages/term/create/request.json");
    }
}

File: message-api-schema-validator/tests/term/term_create_request_test.py
from tests.abstract_schema_validator_test import AbstractSchemaValidatorTest
from unittest import TestCase


class TermCreateRequestTest(AbstractSchemaValidatorTest, TestCase):
    def get_json_schema_file_name(self):
        return "messages/term/create/request_schema.json"

    def test_validate_request(self):
        self.validate_json("messages/term/create/request.json")
